web_external/ImagesGallery/Listing/ImageWall.js

```diff
@@ -23,6 +23,19 @@ const ImageWall = View.extend({
             } else {
                 clickedImage.toggleSelected();
             }
+        },
+
+        'click .isic-images-imageWall-zoom': function (event) {
+            let imageId = $(event.currentTarget).parent().parent().data('imageId');
+            let clickedImage = this.images.get(imageId);
+
+            new ImageFullscreenWidget({ // eslint-disable-line no-new
+                el: $('#g-dialog-container'),
+                model: clickedImage,
+                parentView: this
+            }).render();
```

Comment:
Nit: This `new ImageFullscreenWidget({ ...` could be refactored, since the other `click` handler performs the same operation.

Reply:
I was sort of split on whether to do this, but it's not a bad idea.